core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/kubernetes/reconfigure-rabbitmq-app.yml

```diff
@@ -3,3 +3,70 @@
   include_tasks: utils/patch-statefulset.yml
   vars:
     image_regexp: 'rabbitmq:.*'
+- name: Change rabbitmq stateful set to use {{ image_registry_address }}
+  block:
+    - name: upgrade-master | Get rabbitmq statefulset name
+      environment:
+        KUBECONFIG: /home/{{ admin_user.name }}/.kube/config
+      shell: |-
+        kubectl get statefulsets.apps --all-namespaces -o=jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.spec.template.spec.containers[].image}{"\n"}{end}'|
+        grep -i rabbitmq |
+        awk '{print $1}'
```

Comment:
I would get rid of additional pipe here, pls consider to use here only awk (eg. awk '/rabbitmq/ {print $1}'